app/models/hyku_addons/csv_entry.rb

```diff
@@ -59,11 +78,31 @@ def add_json_fields
       end
     end
 
+    def add_resource_type
+      resource_type_service = HykuAddons::ResourceTypesService.new(model: parsed_metadata['model'].safe_constantize)
```

Comment:
Will this need a safe operator? 

`parsed_metadata['model']&.safe_constantize`

Reply:
I think it should always be there but wouldn't hurt to add and might help with debugging.  I'll get it in the next PR.